src_c/math.c

```diff
@@ -779,6 +781,17 @@ vector_nonzero(pgVector *self)
     return 0;
 }
 
+static PyObject *
+vector_copy(pgVector *self, PyObject *args)
```

Comment:
Here the `PyObject *args` argument is redundant because the copy method is a `METH_NOARGS` anyways